app/controllers/application_controller.rb

```diff
@@ -1,7 +1,17 @@
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
```

Comment:
The GDS SSO uses `helper_method` which is not available in the `API` controller.